Src/Plugins/Security/SEC101_036.MySqlConnectionStringValidator.cs

```diff
@@ -121,25 +121,29 @@ static MySqlConnectionStringValidator()
             string database = fingerprint.Resource;
 
             if (string.IsNullOrWhiteSpace(host) ||
-                string.IsNullOrWhiteSpace(database) ||
                 FilteringHelpers.LocalhostList.Contains(host))
             {
                 return ValidationState.Unknown;
             }
 
-            string connString = $"Server={host}; Database={database}; Uid={account}; Pwd={password}; SslMode=Preferred;";
-            message = $"the '{account}' account was authenticated against database '{database}' hosted on '{host}'";
-
-            if (!string.IsNullOrWhiteSpace(port))
+            bool shouldRetry;
+            string connString;
+            if (!string.IsNullOrWhiteSpace(database))
             {
-                connString += $"Port={port}";
-            }
+                connString = $"Server={host}; Database={database}; Uid={account}; Pwd={password}; SslMode=Preferred;";
+                message = $"the '{account}' account was authenticated against database '{database}' hosted on '{host}'";
```

Comment:
>the '{account}' account was authenticated against database '{database}' hosted on '{host}'"; [](http://example.com/codeflow?start=28&length=92)

Doesn't this message actually get returned when we *didn't* authenticate?

an attempt to authenticate the  '{account}' for database '{database}' hosted on '{host}' did not succeed";

Reply:
no. we are using our helpers that override the message. so, this would only appear when we have an authorized connection.

Reply:
ok!